package.json

```diff
@@ -44,6 +44,7 @@
     "fix-path": "2.1.0",
     "gatsby-cli": "1.1.58",
     "ps-tree": "1.1.0",
+    "redux-saga": "0.16.0",
```

Comment:
Ah I just noticed; this should be a `devDependency`.

While it seems counterintuitive that we're moving all the stuff we actually use to `devDependencies`, the rationale is that we don't want to bundle them in our dist bundles. We only want to ship the `node_modules` that the Node app needs to do its job at runtime, not the ones bundled by Webpack for "browser" usage.